### PR TITLE
Update errors on VIC form

### PR DESCRIPTION
**Pull request by jbalboni**
This updates some of the errors from the error spreadsheet. Peggy is going to take another pass, which should catch any remaining ones.

I also fixed a bug I found in the screen reader upload path where we weren't checking file types.

screen shot 2018-02-13 at 4 32 36 pm
screen shot 2018-02-13 at 4 32 03 pm
screen shot 2018-02-13 at 4 31 18 pm

### CODE REVIEW

File: src/js/vic-v2/config/form.js [hunk 1]
```diff
@@ -74,6 +74,9 @@ const formConfig = {
               'ui:title': 'Gender',
               'ui:options': {
                 labels: genderLabels
+              },
+              'ui:errorMessages': {
+                required: 'Please select the response that most closely aligns with how you identify'
```
Comment (rianfowler) [inline src/js/vic-v2/config/form.js]: Should this end with a period?

**Review by rianfowler — APPROVED** [review]

**Review by peggygannon — COMMENTED** [review]
@jbalboni I made some tweak to the error messages in the files, but I didn't see the errors that are in the screenshots in this PR:
--Re-upload Photo. change this to Upload Photo Again
--add period to end of sentence: Please select the response that most closely aligns with how you identify. 
--this last message should read: You're signed out of your account. Please sign in again to continue your application for a Veteran ID Card. 
--Sign in button should be Sign In (cap I)

**Comment by jbalboni** [comment]
@peggygannon Generally the validation errors we display above the form inputs don't have periods. Do you want to add one for the gender message?

**Comment by peggygannon** [comment]
@jbalboni @rianfowler Yes, please add a period to the end of this sentence. I'll be recommending adding punctuation to end of these message when the sentence is longer.

File: src/js/vic-v2/config/form.js (hunk 1 above)
Comment (cehsu) [reply src/js/vic-v2/config/form.js]: This seems like it is also serving the purpose of a description, in addition to that of an error message.

**Review by cehsu — APPROVED** [review]

**Pull request merged**